_resources/port1.0/group/python-1.0.tcl

```diff
@@ -291,20 +291,10 @@ proc python_get_defaults {var} {
         prefix {
             global build_arch frameworks_dir
```

Comment:
No longer using build_arch.

Reply:
so it should just be `global  frameworks_dir`